readme.md

```diff
@@ -19,9 +19,9 @@
   <p align="center">
     <a href="#demo">View Demo</a>
     ·
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Report Bug</a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aissue+is%3Aopen+label%3Abug">Report Bug</a>
     ·
-    <a href="https://github.com/anuraghazra/github-readme-stats/issues">Request Feature</a>
+    <a href="https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aissue+is%3Aopen+label%3Afeature">Request Feature</a>
```

Comment:
Hi @rjoydip thanks for the PR, i just realized it's would be better if the URL is this https://github.com/anuraghazra/github-readme-stats/issues/new/choose

Reply:
Done